apps/gateway-admin/app/globals.css

```
@@ -91,8 +91,8 @@
 }
 
 @theme inline {
-  --font-sans: 'Geist', 'Geist Fallback';
-  --font-mono: 'Geist Mono', 'Geist Mono Fallback';
+  --font-sans: 'Segoe UI', 'Helvetica Neue', Arial, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)